pkgs/development/python-modules/aiohttp-socks/default.nix

```
@@ -0,0 +1,21 @@
+{ lib, fetchPypi, buildPythonPackage, pythonOlder, aiohttp }:
+
+buildPythonPackage rec {
+  pname = "aiohttp_socks";
+  version = "0.2.2";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0473702jk66xrgpm28wbdgpnak4v0dh2qmdjw7ky7hf3lwwqkggf";
+  };
+
+  propagatedBuildInputs = [ aiohttp ];
+
+  disabled = pythonOlder "3.5.3";
+
+  meta = {
+    description = "SOCKS proxy connector for aiohttp.";
```

```suggestion
    description = "SOCKS proxy connector for aiohttp";
```
See https://nixos.org/nixpkgs/manual/#sec-standard-meta-attributes.